modules/ipi-install-bmc-addressing.adoc

```diff
@@ -64,59 +66,99 @@ platform:
           password: <password>
           disableCertificateVerification: True
 ----
+
 [discrete]
 == Redfish APIs
 
-Several redfish API endpoints are called onto your BCM when using the bare-metal installer-provisioned infrastructure.
+The installation program calls several Redfish API endpoints on the BMC when using installer-provisioned infrastructure on bare metal.
 
 [IMPORTANT]
 ====
-You need to ensure that your BMC supports all of the redfish APIs before installation.
+Ensure that your BMC supports all of the Redfish APIs before installation.
 ====
 
-List of redfish APIs::
-* Power on
+.Procedure
+
+. Set the BMC user name by running the following command:
++
+[source,terminal]
+----
+$ export USER=<username> <1>
+----
+<1> Replace `<username>` with the name of the BMC user.
+
+. Set the BMC password by running the following command:
++
+[source,terminal]
+----
+$ export PASS=<password> <1>
+----
+<1> Replace `<password>` with the password of the BMC user.
+
+. Set the IP address of the BMC by running the following command:
++
+[source,terminal]
+----
+$ export SERVER=<ip_address> <1>
+----
+<1> Replace `<ip_address>` with the IP address of the BMC.
+
+. Set the ID of the system by running the following command:
++
+[source,terminal]
+----
+$ export SystemID=<system_id> <1>
+----
+<1> Replace `<system_id>` with the system ID. For example, `System.Embedded.1` or `1`. See the following vendor-specific BMC sections for details.
```

Comment:
This callout is needed as it provides useful input details.

Reply:
Agreed.